pkg/network/encoding/marshal/usm_encoders_windows.go

```diff
@@ -0,0 +1,22 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+//go:build windows && npm
+
+package marshal
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/network"
+)
+
+func initializeUSMEncoders(conns *network.Connections) []usmEncoder {
+	encoders := make([]usmEncoder, 0, 1)
+
+	if encoder := newHTTPEncoder(conns.HTTP); encoder != nil {
```

Comment:
why is only HTTP added for Windows? I don't see any platform tags/cases in the original code.

Reply:
The windows version of USM supports only the http protocol
all other protocols are linux only

Reply:
> I don't see any platform tags/cases in the original code.

That's something I'm planning to change, this is one of the preliminary PRs to allow that :)